Matveibos/hometask-1/hello_world.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
```

Comment:
This won't make the file executable.
Try to run $ ./hello_world.rb in the hometask folder.